ManatiCR/dccr
# Cards components improvements

Closes #82 

## How to test
We need to test the `Card` and `Card Deck` components.

- Add a `Card` component to a basic page:
  - [x] There should be a field `Background color` with four options for the card background, this options are Blue, Orange, Light Blue and Gray.
  - [x] Whenever you change the background color, when you see the element in the node display, you should get the color of the text to match with it, so this is the mapping:
    - Blue background: white text.
    - Orange background: white text.
    - Light blue background: blue text.
    - Gray background: black text.
- Add a `Card Deck` component to a basic page:
  - [x] There should be a field for `Amount of columns in the grid` and it should be a select list where you can choose between 2 and 5 columns.
  - [x] When you view the node display, the cards should be shown correctly respecting the background color, text color and the amount of columns in the grid specified. Please test with all the number of columns allowed.

Add any additional required notes
